core/db_client.py

```diff
@@ -49,6 +49,7 @@ def _get_labels(self):
 			try:
 				return pd.read_sql_query(query, self.db.engine)
 			except Exception as e:
+				print(e)
```

Comment:
Please log this exception instead.

Reply:
Forgot to delete actually, but sure I'll actually just log it.